debian/rules

```diff
@@ -11,6 +11,7 @@ configure_providers = \
 	--enable-verbs \
 	--enable-rxm \
 	--enable-shm \
+	--enable-psm3 \
```

Comment:
```
[2023-11-14T17:17:06.444Z] configure:24882: checking for linux/idxd.h
[2023-11-14T17:17:06.444Z] configure:24882: result: no
[2023-11-14T17:17:06.444Z] configure:25070: psm3 provider: disabled
[2023-11-14T17:17:06.444Z] configure:25107: WARNING: psm3 provider was requested, but cannot be compiled
```

Reply:
ok I'm trying something now but I might want to keep it disabled on debian builds...